api/managementpb/backup/backups.proto

```diff
@@ -176,6 +187,17 @@ message RemoveScheduledBackupRequest {
 
 message RemoveScheduledBackupResponse {}
 
+message GetMongoPITRRangesRequest {}
```

Comment:
shouldn't be there something like service id?

Reply:
I will remove that method for now. There will separate ticket for that.